.github/workflows/demo-mock-bank-e2e.yml

```
@@ -0,0 +1,129 @@
+      - name: Drop local replace directives (CI uses published modules)
+        run: |
+          go mod edit -dropreplace github.com/cordum/cordum/sdk || true
+          go mod tidy
+
+      - name: Build cordumctl
+        run: |
+          mkdir -p ./bin
+          go build -o ./bin/cordumctl ./cmd/cordumctl
+
+      - name: Generate API key + Redis password
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:50: Double quote to prevent globbing and word splitting

(shellcheck)